src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -157,6 +167,21 @@ def launch_fluent(
         default is ``True``. You can stop and start the streaming of the
         Fluent transcript subsequently via the method calls, ``transcript.start()``
         and ``transcript.stop()`` on the session object.
+    exposure : FluentExposure or str, optional
+        Exposure of Fluent. Options are either the values of the ``FluentExposure`` enum
+        or any of ``"no_gui_or_graphics"``, ``"no_gui"``, ``"hidden_gui"``,
+        ``"no_graphics"`` or ``"gui"``. The default is ``FluentExposure.HIDDEN_GUI`` in
+        Windows and ``FluentExposure.NO_GUI`` in Linux. ``"no_gui_or_graphics"`` and
+        ``"no_gui"`` exposure are supported in Windows only for Fluent version 2024 R1
+        or later.
```

Comment:
I think people looking for this option will probably be thinking something along the lines of "I want to hide the Fluent interface" or those more familiar "I want to change the interface launch mode of Fluent"

What about calling it `interface` rather than `exposure`? 
```suggestion
    interface: FluentInterface or str, optional
        Interface launch mode of Fluent. Options include the values of the ``FluentInterface`` 
        enum, as well as ``"no_gui_or_graphics"``, ``"no_gui"``, ``"hidden_gui"``,
        ``"no_graphics"`` and ``"gui"``. The default is ``FluentInterface.HIDDEN_GUI`` in
        Windows and ``FluentInterface.NO_GUI`` in Linux. ``"no_gui_or_graphics"`` and
        ``"no_gui"`` interface are supported in Windows only for Fluent version 2024 R1
        or later.
```

Reply:
@raph-luc I feel `interface` can be bit technical for some users. As this is purely a Fluent side option, we can alternatively call it `fluent_ui` so it precisely means what it is for (both `interface` and `exposure` have a broad meaning).

Reply:
@mkundu1 thanks for the explanation, I think that "UI" or "interface" are similarly technical, with "UI" (User Interface) being a bit more technical than "interface". 

Regarding `fluent_ui`, we are already in the `launch_fluent` function, so the `fluent_` prefix is probably unnecessary.

I do personally think that "exposure" is a significantly more technical/obscure term than "interface", hence my suggestion. I guess "window" would be even less technical than "interface", but not sure if it would be appropriate here.

Is there someone from the UX or doc teams that we could briefly ask which one they would suggest?

cc @PipKat @seanpearsonuk

Reply:
I prefer `FluentVisibility` over `FluentDisplayMode`. `FluentExposure` is good although some users could easily think it refers to API exposure (Pro, Enterprise etc), so it would need to be expanded to `FluentInterfaceExposure` (still ambiguous) and therefore `FluentGUIExposure` or the less blatant `FluentUIExposure`. I would be unsure of the meaning of `FluentInterface`.

Reply:
@seanpearsonuk if `FluentInterface` is too broad, what about just `FluentUI` with a respective `ui` argument for `launch_fluent()`, similar to what @mkundu1 proposed earlier? I would argue there no need to add an `Exposure` suffix.